src/Account.java

```diff
@@ -1,2 +1,32 @@
+import java.util.ArrayList;
+import java.util.List;
+
 public class Account {
+    private String username;
+    private final String password;
+    private List<Deck> decks;
```

Comment:
`decks` should have a getter method so that we can access it from outside the class.

Reply:
yeah thats what i was thinking as well - that should be implemented within AccountInteractor?

Reply:
I think the getter method is fine only being in `Account`, since none of the other Interactors duplicate any getter or setter methods.

Reply:
sounds good